src/cli/tc.rs

```diff
@@ -60,51 +59,81 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
```

Comment:
actually name `file_path` is confusing since it specified a folder. Could you change it?

Reply:
also, as another improvement do we need this option to be required in cli? default to current dir could be sane option

Reply:
both solved